internal/scheduler/jobs/patch_anchor_platform_transactions_job.go

```diff
@@ -0,0 +1,47 @@
+package jobs
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/anchorplatform"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/data"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/services"
+)
+
+const (
+	PatchAnchorPlatformTransactionsJobName            = "patch_anchor_platform_transactions"
+	PatchAnchorPlatformTransactionsJobIntervalSeconds = 10
+)
+
+type PatchAnchorPlatformTransactionsJob struct {
+	service *services.PatchAnchorPlatformTransactionService
+}
+
+func NewPatchAnchorPlatformTransactionsJob(apAPISvc anchorplatform.AnchorPlatformAPIServiceInterface, models *data.Models) *PatchAnchorPlatformTransactionsJob {
+	svc, err := services.NewPatchAnchorPlatformTransactionService(apAPISvc, models)
+	if err != nil {
+		log.Fatalf("error instantiating anchor platform service: %v", err)
```

Comment:
```suggestion
		log.Fatalf("instantiating anchor platform service: %v", err)
```

Reply:
Fatal errors are not a good practice, but I can see there's a whole architectural mistake related to this that didn't support the default error handling.